code/src/components/TodoList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import { TodoInput } from "./TodoInput";
+import { TodoItem } from "./TodoItem";
+
+export const TodoList = () => {
+  const list = useSelector(store => store.todos.list);
+  return (
+    <div className="todo-list">   
+      <TodoInput />
+      {list.items.map((item, index) => (
+      <TodoItem key={index} itemIndex={index}></TodoItem>
```

Comment:
you could pass along the whole item here instead of just the index and then you don't have to get it again from the global state.